stone/target/js_client.py

```diff
@@ -44,7 +45,7 @@ class JavascriptClientGenerator(CodeGenerator):
     cmdline_parser = _cmdline_parser
 
     # Instance var of the current namespace being generated
-    cur_namespace = None  # type: ignore # TODO: What is the type of this thing?!
+    cur_namespace = None  # type: ApiNamespace
```

Comment:
Does this need to be marked `Optional[ApiNamespace]` or does mypy infer that from the declaration?